### DISCUSSION TITLE
Is It Possible to Add a Watermark in AR Mode?

### DISCUSSION THREAD

**Discussion by NadimSKanaan** [google/model-viewer#1836] [Q&A]
I am using Scene Viewer and Quick Look as AR modes. Is it possible to add a watermark in the top left/right corner for branding purposes?

**Comment by elalish** [comment]
Not that I'm aware of (though they do tend to expand their APIs from time to time). This is why we default to WebXR mode, because we can use DOM Overlay so that all child elements show up just the way you want.
